doc/ADMIN.md

```diff
@@ -1,3 +1,11 @@
-If you wish, you can add custom configurations inside `__INSTALL_DIR__/config.local.php`.
+If you wish, you can add custom configurations inside `__DATA_DIR__/data/config.local.user.php`.
+
+Use the `__DATA-DIR__/data` folder if you want to store documents elsewhere than in the database. Example in `__DATA_DIR__/data/config.local.user.php` file
```

Comment:
Isn't there a typo here: __DATA-DIR__/data? Shouldn't it rather be __DATA_DIR__/data (with the underscore)